bots/images/scripts/fedora.setup

```diff
@@ -27,6 +27,11 @@ if [ $(uname -m) = x86_64 ]; then
     HAVE_KUBERNETES=1
 fi
 
+HAVE_LIBVIRT_DBUS=
+if [ "${IMAGE:(-2)}" -gt 27 ]; then
```

Comment:
This will probably no work for `fedora-testing`

Reply:
Oh good catch, I 'll repost.

Reply:
Let me know if you can think of some nicer regexp to match these images names.